### ISSUE TITLE
why can't I change the invalid batch_size

### ISSUE THREAD

**Issue by KelvinHuang666** [aditya-vora/FCHD-Fully-Convolutional-Head-Detector#40]
when I change  the invalid batch_size=128 or other number !=1,
RuntimeError: invalid argument 0: Sizes of tensors must match except in dimension 0. Got 861 and 811 in dimension 3 at /pytorch/aten/src/TH/generic/THTensorMoreMath.cpp:1307

**Comment by LeonardNiu** [comment]
> 
> 
> when I change the invalid batch_size=128 or other number !=1,
> RuntimeError: invalid argument 0: Sizes of tensors must match except in dimension 0. Got 861 and 811 in dimension 3 at /pytorch/aten/src/TH/generic/THTensorMoreMath.cpp:1307

I've met the same problem, have you already solved it?

**Comment by DANNALI35** [comment]
yes, i encountered the same issue. after i pad the label bbox to the same size, the error has gone, but i finally got the following error below, it's raised from **trainer.py**.
` raise ValueError('Currently only batch size 1 is supported.')    `

i believe this project is mainly for demo purpose where batch size= 1 works fine,  i guess i just have to wait longer for the training process.

BTW, brilliant work with visdom, i cannot believe it's realtime and interactive!
